examples-extra/docs_example/src/generatedNoCheck/ontology/actions/moveOffice.ts

```diff
@@ -36,21 +35,39 @@ export type ActionDef$moveOffice$Params = {
   };
 };
 
-// Represents the runtime arguments for the action
-export type moveOffice$Params =
-  | NOOP<OsdkActionParameters<ActionDef$moveOffice$Params>>
-  | NOOP<OsdkActionParameters<ActionDef$moveOffice$Params>>[];
+/**
+ * Update an office's physical location
+ */
+export interface ActionParams$moveOffice {
+  /**
+   * The office's new physical address (not necessarily shipping address)
+   */
+  readonly newAddress?: ActionParam.PrimitiveType<'string'>;
+  /**
+   * The maximum seated-at-desk capacity of the new office (maximum fire-safe capacity may be higher)
+   */
+  readonly newCapacity?: ActionParam.PrimitiveType<'integer'>;
+
+  readonly officeId: ActionParam.PrimitiveType<'string'>;
+  /**
+   * A list of all office names
+   */
+  readonly officeNames?: ReadonlyArray<ActionParam.PrimitiveType<'integer'>>;
+}
+
+/**
+ * @deprecated Use `ActionParams$moveOffice`
+ */
+export type moveOffice$Params = ActionParams$moveOffice | ReadonlyArray<ActionParams$moveOffice>;
```

Comment:
Little confused by these, these are using your new types right, just unioning the array or singular parameters case. Why do we want to deprecate this?

Reply:
Like I know its the old identifier, but seems fine because it's using you're new types?

Reply:
I didn't want to break the internal code base right out. This should  make it easier for us to upgrade to an alpha and regex fix the internal usages in separate steps.